### ISSUE TITLE
[bash-completion] colon induces `sed` errors in the `scp` completion

### ISSUE THREAD

**Issue by Lun4m** [akinomyoga/ble.sh#123]
**OS**: Ubuntu 21.04
**Terminal**: Gnome Terminal 3.38.1
**ble**: 0.4.0-devel3+a034c91 (noarch)
**Bash**: 5.1.4(1)-release (x86_64-pc-linux-gnu)

I don't know if it's a bug, but I couldn't find any info in the README.md, so here I am.
I'm trying to use `scp` to copy files from a remote host, but when I type:
```bash
scp -r host:
```
I get the following:
```bash
scp -r host:sed: option requires an argument -- 'e'
```
I don't have a `.blerc` and this doesn't happen if `ble.sh` is detached.
Also, as far as I can tell this problem is specific to `scp`, i.e. if I use `:` while typing anything else, then `sed` won't be executed.

**Comment by akinomyoga** [comment]
Thank you for the report! OK, I could reproduce the problem locally. It happens when `bash_completion` is loaded.

It happens without ble.sh when `shopt -s nullglob` is set.

<pre>
$ bash --norc                                       # (start plain Bash session)
$ shopt -s nullglob                                 # (set nullglob)
$ source /usr/share/bash-completion/bash_completion # (load bash-completion)
$ scp host:<kbd>TAB</kbd>
</pre>

It seems that `shopt -s nullglob` is somehow turned on while running `ble.sh`, and then the problem is caused.

**Comment by akinomyoga** [comment]
- I have opened a PR [`bash-completion` \#556](https://github.com/scop/bash-completion/pull/556) to fix the error messages. But it will take time until the fix is applied to downstream distributions.
- I have added a fix 899c11428ce6ff94e95cf1977600b13011ec6c00 for the problem that `nullglob` is turned on unexpectedly.
- I also added a workaround d117973afad2ca0e06c360d7e84fe5fd5549afbc for `bash-completion` v2.11.
